userspace/falco/app/actions/helpers_interesting_sets.cpp

```diff
@@ -116,7 +160,15 @@ static void select_event_set(falco::app::state& s, const libsinsp::events::set<p
 
 static void select_syscall_set(falco::app::state& s, const libsinsp::events::set<ppm_event_code>& rules_event_set)
 {
-	s.selected_sc_set = libsinsp::events::event_set_to_sc_set(s.selected_event_set);	
+	s.selected_sc_set = libsinsp::events::event_set_to_sc_set(s.selected_event_set);
+
+	if (!s.selected_sc_set.empty())
```

Comment:
This needs the outstanding fixes from https://github.com/falcosecurity/libs/pull/889.

Reply:
I think the ones required here have been isolated in https://github.com/falcosecurity/libs/pull/921 and already merged.